frontend/js/src/explore/lb-radio/Playlist.tsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import * as React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { faCog, faFileExport } from "@fortawesome/free-solid-svg-icons";
+import { faSpotify } from "@fortawesome/free-brands-svg-icons";
+import GlobalAppContext from "../../utils/GlobalAppContext";
+import PlaylistItemCard from "../../playlists/PlaylistItemCard";
+
+type UserFeedbackProps = {
+  feedback: string[];
+};
+
+type PlaylistProps = {
+  playlist?: JSPFPlaylist;
+  title: string;
+  onSavePlaylist: () => void;
+  enableOptions: boolean;
+  onSaveToSpotify: () => void;
+  onExportJSPF: () => void;
```

Comment:
I personally want to refactor these playlist export functions into a reusable menu component that we can plunk into every page that deals with playlists